excretor/src/models.rs

```diff
@@ -50,6 +73,18 @@ impl User {
             is_bot: db_message_and_user.is_bot,
         }
     }
+    pub fn from_db_reply(db_reply: &dbmodels::DBReply) -> Self {
```

Comment:
Can this be implemented using the `From<T>` trait?

Reply:
Donee